turbo.json

```diff
@@ -42,6 +43,11 @@
         "NEXT_PUBLIC_ALGOLIA_APP_ID",
         "SITE_URL"
       ],
+      "outputs": [".next/**", "dist/**"]
+    },
+    "@hive/migrations#build": {
+      "dependsOn": ["^build"],
+      "inputs": ["src/**/*"],
```

Comment:
turbo treats everything within the package as inputs

Reply:
oh cool, fixed! :)